.eslintrc.yml

```diff
@@ -1,9 +1,14 @@
 env:
   es2021: true
   node: true
+parser: "@babel/eslint-parser"
 parserOptions:
-  ecmaVersion: 12
+  requireConfigFile: false
+  ecmaVersion: latest
```

Comment:
latest might be a bit dangerous, because it changes constantly
could we fix it to a specific number?